futures-util/src/compat/compat01as03.rs

```diff
@@ -95,6 +120,168 @@ impl<St: Stream01> Stream03 for Compat01As03<St> {
     }
 }
 
+/// Converts a futures 0.1 Sink object to a futures 0.3-compatible version
+#[derive(Debug)]
+#[must_use = "futures do nothing unless polled"]
```

Comment:
This isn't a future.

```suggestion
#[must_use = "sinks do nothing unless polled"]
```

Reply:
You are right, Ill add this change!